server/migrations/20170321114511-update-usermessageevents.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.addColumn('UserMessageEvents', 'fromUserId', {

```

Comment:
fromUserId and toUserId are immutable properties of the Message.  They should not be replicated here in the UserMessageEvent.